### PR TITLE
Display administrator information on global settings area

### PR DESCRIPTION
**Pull request by AviGawande**
This PR fixes and closes issue #138

## Summary by Sourcery

This pull request introduces a new feature to display and manage administrator information in the global settings area. It includes a new form and view for capturing admin name and email, and updates the global navigation to link to this new page.

- **New Features**:
    - Added a new form to capture administrator information, including admin name and email, in the global settings area.
    - Introduced a new view to handle the display and submission of the administrator information form.
- **Enhancements**:
    - Updated the global navigation to include a link to the new administrator information page.

### CODE REVIEW

**Comment by mariobehling** [comment]
This is apparently copied from Pretalx without required changes.

**Comment by AviGawande** [comment]
There are system specific changes in the code for evetyay tickets

File: src/pretix/control/views/global_settings.py [hunk 1]
```diff
@@ -83,3 +83,20 @@ class RefundDetailView(AdministratorPermissionRequiredMixin, View):
...
+        form.save()
+        messages.success(self.request, _('Your changes have been saved.'))
+        return super().form_valid(form)
+
+    def form_invalid(self, form):
+        messages.error(self.request, _('Your changes have not been saved, see below for errors.'))
+        return super().form_invalid(form)
+
+    def get_success_url(self):
+        return reverse('control:admin.information')
```
Comment (hongquan) [inline src/pretix/control/views/global_settings.py]: Please configure your editor to automatically add one blank line at the end of file.

**Review by hongquan — CHANGES_REQUESTED** [review]

File: src/pretix/control/views/global_settings.py (hunk 1 above)
Comment (AviGawande) [reply src/pretix/control/views/global_settings.py]: sure,configured the Editor.

**Comment by mariobehling** [comment]
@sourcery-ai review

File: src/pretix/control/templates/pretixcontrol/global_info.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        </ul>
+    {% endif %}
+
+
+
+<!-- Reference for adding the administrator information that should be displayed on the global settings of the eventyay-tickets  -->
+
+<h2 class="mt-4">{% translate "Links" %}</h2>
+<ul>
+    <li><a target="_blank" rel="noopener" href="https://eventyay.com">{% translate "Checkout Our website" %}</a></li>
```
[the remark this answers is not in the document]
Comment (AviGawande) [reply src/pretix/control/templates/pretixcontrol/global_info.html]: fixed changes

File: src/pretix/control/templates/pretixcontrol/global_info.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        <li>{% translate "Plugins" %}: {% if settings.PLUGINS %}<ul>{% for plugin in settings.PLUGINS %}<li>{{ plugin }}</li>{% endfor %}</ul>{% else %}–{% endif %}</li>
+        <li>{% if settings.ADMINS %}{% translate "On errors, emails will be sent to:" %}
+            {% for admin in settings.ADMINS %}
+                <code>{{ admin.0 }}</code>{% if not forloop.last %},{% endif %}
+            {% endfor %}
+        {% else %}{% translate "On errors, no emails will be sent." %}{% endif %}
+        </li>
+    </ul>
+
+    <h3 class="mt-4">Redis</h3>
```
[the remark this answers is not in the document]
Comment (AviGawande) [reply src/pretix/control/templates/pretixcontrol/global_info.html]: fixed

**Pull request closed without merging**